src/PrestaShopBundle/Resources/config/services/adapter/meta.yml

```diff
@@ -16,6 +16,7 @@ services:
             - '@prestashop.core.hook.dispatcher'
             - '@validator'
             - '@=service("prestashop.adapter.legacy.configuration").get("PS_LANG_DEFAULT")'
+            - '@=service("prestashop.adapter.meta.data_provider").getAvailablePages()'
```

Comment:
Consider injecting service instead, data will be cached if you only inject result which may cause unwanted side effects.